scripts/tosdr.js

```diff
@@ -50,12 +50,27 @@ function getSitePoints (sites) {
         for (pointName in pointsData) {
             let point = pointsData[pointName]
             let pointCase = point.tosdr.case
+            let score = point.tosdr.score || 0
             if (!pointCase) continue
-            
+
+            // standardize case (some of them start with caps)
+            pointCase = pointCase.toLowerCase()
+            // standardize score (some of them come as strings)
+            score = parseInt(score, 10)
```

Comment:
There were a couple of inconsistencies that we need to deal with here. "device fingerprinting" comes from TOSDR with a capitalized "D", and I saw some string values for `score`, which led to them being concatenated below, e.g. "050".